### ISSUE TITLE
Unable to zoom images with touchpad / trackpoint

### ISSUE THREAD

**Issue by baltitenger** [Nheko-Reborn/nheko#1448]
### Describe the bug

Scrolling with the mouse zooms as expected, but neither "scrolling" nor pinching on the touchpad does anything while an image is opened.

### To Reproduce

1. Click on an image in the timeline
2. Try to zoom in using the touchpad or track point

### Version

0.11.3

### Operating system

Linux

### Installation method

Some repository (AUR, homebrew, distribution repository, PPA, etc)

### Qt version

5.15.9+kde+r7

### Desktop Environment

sway

### Did you use profiles?

- [ ] Profiles used?

**Comment by deepbluev7** [comment]
Hm, I can't reproduce this...

**Comment by deepbluev7** [comment]
I don't really know where to go from here, so I might just let this issue sit a while until I have a good idea. It might be helpful to explain your hardware in more detail. The master branch might also have improved it by switching to qt6 (but there are regressions). I am assuming normal scrolling works just fine?

**Comment by baltitenger** [comment]
Yeah, everything else is fine but whatever I do with either the touchpad or the trackpoint does nothing on the image view.

As for hardware, this is a thinkpad T490, and this is what libinput says about the devices in question:
```
Device:           SynPS/2 Synaptics TouchPad
Kernel:           /dev/input/event9
Group:            7
Seat:             seat0, default
Size:             100x68mm
Capabilities:     pointer gesture
Tap-to-click:     disabled
Tap-and-drag:     enabled
Tap drag lock:    disabled
Left-handed:      disabled
Nat.scrolling:    disabled
Middle emulation: disabled
Calibration:      n/a
Scroll methods:   *two-finger edge 
Click methods:    *button-areas clickfinger 
Disable-w-typing: enabled
Disable-w-trackpointing: enabled
Accel profiles:   flat *adaptive custom
Rotation:         n/a

Device:           TPPS/2 Elan TrackPoint
Kernel:           /dev/input/event16
Group:            8
Seat:             seat0, default
Capabilities:     pointer 
Tap-to-click:     n/a
Tap-and-drag:     n/a
Tap drag lock:    n/a
Left-handed:      disabled
Nat.scrolling:    disabled
Middle emulation: disabled
Calibration:      n/a
Scroll methods:   *button
Click methods:    none
Disable-w-typing: n/a
Disable-w-trackpointing: n/a
Accel profiles:   flat *adaptive custom
Rotation:         n/a
```

I will try master later

**Comment by deepbluev7** [comment]
So you are using the touchpad edge for scrolling? Does it work if you use the whole touchpad for scrolling or if you just use the edge?

**Comment by deepbluev7** [comment]
Ah, wait, I guess the * is the enabled method and the option without the * is the disabled alternative?

**Comment by baltitenger** [comment]
I believe so, yes. I use two fingers for scrolling.

**Comment by deepbluev7** [comment]
I have pretty much the same touchpad on my laptop and it seems to work fine there on Sway, so it is a bit curious. If anybody has an idea, I would be welcome to hear it :D

**Comment by deepbluev7** [comment]
Hm, maybe you are affected by https://bugreports.qt.io/browse/QTBUG-112394 which I added a workaround for to the master branch just now, because it seems to be worse on qt6.

**Comment by baltitenger** [comment]
Hmm, tried master now and it does appear to work though now it's broken in a different way :P
Now anything I do on the image view is like 0.5 fps for some reason. (also the theme appears to be broken too but that might just be an issue with my config)

**Comment by baltitenger** [comment]
hm, and I take it I can't just downgrade back to the version I was using before...

**Comment by baltitenger** [comment]
Do you think this can be salvaged still? After downgrade, it dies with
```
terminate called after throwing an instance of 'lmdb::runtime_error'
  what():  mdb_dbi_open: Permission denied
Error: signal 6:
```

**Comment by LorenDB** [comment]
That is a known issue that currently hasn't been fixed. I have no idea if we will actually implement a fix or not, since hopefully there will be a new release in the not-too-terribly-distant future.
